libs/components/ag-grid/src/lib/modules/ag-grid/ag-grid.service.spec.ts

```diff
@@ -235,10 +229,7 @@ describe('SkyAgGridService', () => {
     });
 
     it('should unsubscribe from the theme service when destroyed', () => {
-      const overrideOptions = { gridOptions: {} };
-      let gridOptions = agGridService.getGridOptions(overrideOptions);
-
-      expect(gridOptions.rowHeight).toBe(38);
+      expect(agGridService.getHeaderHeight()).toBe(37);
```

Comment:
I realize this was an existing test already but why are we checking rowHeight in "should unsubscribe from the theme service when destroyed"?

Reply:
Might've been a coverage thing, but I like that we have a test for the style so I'm not going to pull it out.

Reply:
Oh I wasn't suggesting pulling it out of testing entirely--moreso renaming the test or, if the style is not relevant to anything else in the test, giving the styles their own separate test.

Reply:
After thinking about it, the visual tests are good enough for testing the CSS. I'm going to leave this as-is, and the test can be removed when `getHeaderHeight` gets removed.